lib/sqlalchemy/sql/elements.py

```diff
@@ -2370,7 +2411,9 @@ def bindparams(
         return self
 
     @util.preload_module("sqlalchemy.sql.selectable")
-    def columns(self, *cols, **types):
+    def columns(
+        self, *cols: ColumnElement[Any], **types: TypeEngine[_T]
+    ) -> util.preloaded.sql_selectable.TextualSelect:
```

Comment:
Line 2546: mypy expects `List[ColumnClause[Any]]` for `TextualSelect.__init__` - as per recently merged `sql.selectable` annotations. However, a `columns()` description says its `*cols` argument is "a series of `_expression.ColumnElement` objects, typically". Not sure which is the best way to resolve this.